<?xml version="1.0"?>
<launch>
	<node name="random_walk" pkg="random_walk" type="random_walk">
        <remap from="cmd_vel" to="husky/cmd_vel" />
        <remap from="base_scan/scan" to="scan" />
		<param name="min_dist" type="double" value="0.8"/>
		<param name="view_angle" type="int" value="90"/>
		<param name="robot_width" type="double" value="0.9"/>
		<param name="linear_vel" type="double" value="0.25"/>
		<param name="angular_vel" type="double" value="0.25"/>
		<param name="sensor_frame_id" type="str" value="/camera_link"/>
		<param name="real_flag" type="bool" value="1"/>
	</node>
</launch>
